src/components/defaultLayout/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Account from '../account';
+import PrivateRoute from '../privateRoute';
+
+const DefaultLayout = ({ component: Component, ...rest }) => (
+  <PrivateRoute {...rest} component={ matchProps => (
+    <main>
+      <Account />
```

Comment:
We don't need to render Account component and Link components every time that we change the route.They are the same in forging and voting and transaction and rendering them over and over is useless and it will decrease performance of our application .So, we must use nesting routes to solve this problem.why do you remove nested routes?